schema.sql

```diff
@@ -7,9 +7,43 @@ CREATE TABLE animals(
   escape_attempts    INT,
   neutered     boolean,
   weight_kg    REAL,
+  species_id INT references species(id),
+  owners_id INT references owners(id),
+  CONSTRAINT fk_species
+      FOREIGN KEY(species_id) 
+	  REFERENCES species(id)
+      ON DELETE SET NULL,
+  CONSTRAINT fk_owners
+      FOREIGN KEY(owners_id) 
+	  REFERENCES owners(id)
+      ON DELETE SET NULL,
   PRIMARY KEY(id)
```

Comment:
- [ ] When I run the command to create the `animals` table in the database, I get an error that the relation `species` and `owners` do not exist. The `animals` table is created before the `species` and `owners` table. To rectify this, you could:

1. Use the ALTER TABLE command to alter the table at the end of the file and add the foreign key to the animals table. The syntax should look like this 
`ALTER TABLE [table_name] ADD [column_name] [data type] REFERENCES [second_table_name](id);` 😄 😎 (Prefered method 👍)

2. Create the `species` and `owners` table before the `animals` table and leave these highlighted lines of code. 💻